gtk/src/subclass/container.rs

```diff
@@ -268,3 +268,19 @@ impl Callback {
         }
     }
 }
+
+pub unsafe trait ContainerClassSubclassExt: ClassStruct {
+    #[doc(alias = "gtk_container_class_handle_border_width")]
+    fn handle_border_width(&mut self) {
+        unsafe {
+            let type_class = self as *mut _ as *mut glib::gobject_ffi::GTypeClass;
+            let widget_class = glib::gobject_ffi::g_type_check_class_cast(
+                type_class,
+                ffi::gtk_container_get_type(),
+            ) as *mut ffi::GtkContainerClass;
```

Comment:
```suggestion
            let widget_class = glib::gobject_ffi::g_type_check_class_cast(
                type_class,
                ffi::gtk_container_get_type(),
            ) as *mut ffi::GtkContainerClass;
```
Is this needed? the check is already implied by implementing the trait only for widgets that implement `ContainerImpl`. I suppose you can directly cast it to as `GtkContainerClass`

Reply:
I copied it from the methods of `WidgetClassSubclassExt`. But yeah, probably not necessary.

Reply:
Yes, not necessary and also not there :)